packages/engine-formula/package.json

```diff
@@ -65,6 +65,8 @@
         "rxjs": ">=7.0.0"
     },
     "dependencies": {
+        "big.js": "^6.2.1",
```

Comment:
Please remove this if is not in your codes.

Reply:
ok